### PR TITLE
Answered two questions. 

### PR DESCRIPTION
**Pull request by reactiongears**
Answered two questions. 

- What is a Promise and how do you use it?
- Is it fine to learn JavaScript as your first language for web development or is it better to start with HTML and CSS

I also removed the quote styling from the last answer that was merged.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,10 +81,50 @@ _Waiting for response_
 # JavaScript   
 
 **What is a Promise and how do you use it?**  
-_Waiting for response_   
+
+A promise in simplest terms is just that. You are calling a function that does dome asyncronous work and and in return it promises to return you a response when it's done.
```
Comment (emmabostian) [inline README.md]:
```suggestion
A promise in simplest terms is just that. You are calling a function that does some asynchronous work and and in return it promises to return you a response when it's done. 
```

File: README.md [hunk 2]
```diff
@@ -81,10 +81,50 @@ _Waiting for response_
...
+```
+
+If you ever get into a situation where you're nesting promised please first take a look a [Promise.all](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all). This is a good way to chain promises.
+
+
 
 **Is it fine to learn JavaScript as your first language for web development or is it better to start with HTML and CSS**   
-_Waiting for response_   
+
+Definitely learn HTML and CSS first. Many take HTML and mostly CSS understanding for granted. But believe me when I say that without a clear understanding of the DOM and CSS you will really struggle as a Web Developer. I have seen bugs get re-opened over and over and over again for the simplest of CSS mistakes. Not being able to cleanly and effectively assemble your layout structure with a stable set of styles can easily turn a 1-day task into a 2-week nightmare and have you wishing you never picked up this career.
```
Comment (emmabostian) [inline README.md]:
```suggestion
Definitely learn HTML and CSS first. Many take HTML and mostly CSS knowledge for granted. Without a clear understanding of the DOM and CSS you will really struggle as a Web Developer. I have seen bugs get re-opened over and over again for the simplest of CSS mistakes. Not being able to cleanly and effectively assemble your layout with a stable set of styles can easily turn a 1-day task into a 2-week nightmare and have you wishing you never picked up this career. 
```

File: README.md [hunk 3]
```diff
@@ -81,10 +81,50 @@ _Waiting for response_
...
+
+
 
 **Is it fine to learn JavaScript as your first language for web development or is it better to start with HTML and CSS**   
-_Waiting for response_   
+
+Definitely learn HTML and CSS first. Many take HTML and mostly CSS understanding for granted. But believe me when I say that without a clear understanding of the DOM and CSS you will really struggle as a Web Developer. I have seen bugs get re-opened over and over and over again for the simplest of CSS mistakes. Not being able to cleanly and effectively assemble your layout structure with a stable set of styles can easily turn a 1-day task into a 2-week nightmare and have you wishing you never picked up this career. 
+
+The big misnomer is that many individuals learn things just enough to be able to Google if they get stuck. CSS isn't one of those things. You can't Google your way out of a CSS bug. Without understanding it, believe me, you can't. The only way is to understand how the cascade works. Once you understand CSS fully then move on to JavaScript.
```
Comment (emmabostian) [inline README.md]:
```suggestion
The big misnomer is that many individuals learn things just enough to be able to Google if they get stuck. CSS isn't one of those things. You can't Google your way out of a CSS bug. The only way is to understand how the cascade works. Once you understand CSS fully then move on to JavaScript. 
```

**Review by emmabostian — CHANGES_REQUESTED** [review]
I posted a couple of requested grammatical changes but other than that this looks great :)

**Comment by emmabostian** [comment]
I actually just realized that someone has already opened a PR for the promises question 😢 Could you please remove your explanation since they're very similar? Apologies! I'm trying to go with the first-come-first-serve method.

File: README.md (hunk 1 above)
Comment (pqt) [reply README.md]: ~Also, there's a second `and`~

Disregard RE https://github.com/emmawedekind/Front-End-FAQ/pull/14#issuecomment-435301616

**Comment by reactiongears** [comment]
Sure Thing! No worries 😁

**Comment by emmabostian** [comment]
Also, could you please fix the merge conflicts? :) I moved questions into separate md files! You're welcome to also open another PR if that's easier for ya.

**Comment by reactiongears** [comment]
Closing this to make a fresh PR after rebasing with the new file structure. @emmawedekind new PR will have your suggested changes from above included.

**Comment by reactiongears** [comment]
I did a rebase and moved the answer after I realized the file structure had changed.

**Comment by reactiongears** [comment]
Conflicts resolved

**Review by emmabostian — APPROVED** [review]
Thanks!

**Comment by reactiongears** [comment]
Always a pleasure! ;-)

**Pull request merged**